blender/arm/logicnode/native/LN_get_system_language.py

```diff
@@ -0,0 +1,13 @@
+from arm.logicnode.arm_nodes import *
+
+class GetSystemLanguage(ArmLogicTreeNode):
+    """Returns the name of the current system."""
```

Comment:
I think the docstring is left over from another node?

Reply:
Yes. Sorry